src/module/linux/netlink.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#define NETLINK_KRF 28
```

Comment:
Potentially duplicated macros?

Reply:
It's in a different section of the source (`src/module` vs `src/krfmesg`). We do a similar thing with `krfctl`.

Reply:
Yep, I see you've also `#define`d it in `krfmesg`, which is fine. But it's also doubly defined here: `NETLINK_MYGROUP` and `NETLINK_KRF` have the same value, so I was wondering whether they can be deduplicated.

Reply:
Ignore this; I see it's both the family and the multicast group.

Reply:
In the future we should probably come up with a common (user and kernelspace) include so that we don't have to dupe these at all, but that's unrelated here.

Reply:
Yeah. We could eventually do something with automake or cmake or another build system to standardize constants like these across all the code sections